File: go/58.length-of-last-word.go
package leetcode

import "strings"

/*
 * @lc app=leetcode id=58 lang=golang
 *
 * [58] Length of Last Word
 */

// @lc code=start
func lengthOfLastWord(s string) int {
    s = strings.TrimSpace(s)

	arr := strings.Split(s, " ")

	return len((arr[len(arr)-1]))
}
// @lc code=end

File: kotlin/58.length-of-last-word.kt
/*
 * @lc app=leetcode id=58 lang=kotlin
 *
 * [58] Length of Last Word
 */

// @lc code=start
class Solution {
    fun lengthOfLastWord(s: String): Int {
        return s.split(' ').for { s -> s.length }.filter { s -> s != 0 }
    }
}
// @lc code=end

